### PR TITLE
Upgrage cartodb extension to 0.18.5

### PR DESCRIPTION
**Pull request by jgoizueta**
New extension revision with some fixes and two new overview strategies.

### CODE REVIEW

File: app/models/user/db_service.rb [hunk 1]
```diff
@@ -518,7 +518,7 @@ def reset_pooled_connections
       # Upgrade the cartodb postgresql extension
       def upgrade_cartodb_postgres_extension(statement_timeout = nil, cdb_extension_target_version = nil)
         if cdb_extension_target_version.nil?
-          cdb_extension_target_version = '0.18.4'
+          cdb_extension_target_version = '0.18.5'
```
Comment (ethervoid) [inline app/models/user/db_service.rb]: why dont we have this as a constant?

**Comment by ethervoid** [comment]
LGTM

**Comment by rafatower** [comment]
:+1:

File: app/models/user/db_service.rb (hunk 1 above)
Comment (jgoizueta) [inline app/models/user/db_service.rb]: ...and why not have it (the default current version) in app_config.yml to avoid deploying just to increase it?

Comment (ethervoid) [inline app/models/user/db_service.rb]: Because Chef doesn't progress all the changes immediately so we're not sure if the change is on all the servers.

**Pull request merged**